hydra-node/test/Hydra/Chain/Direct/Contract/Contest.hs

```diff
@@ -256,6 +279,21 @@ genContestMutation
       , SomeMutation (Just "head value is not preserved") MutateValueInOutput <$> do
           newValue <- genValue
           pure $ ChangeOutput 0 (headTxOut{txOutValue = newValue})
+      , SomeMutation (Just "must push deadline") NotUpdateDeadlineAlthoughItShould . ChangeOutput 0 <$> do
+          let deadline = posixFromUTCTime healthyContestationDeadline
+          -- Here we are replacing the contestationDeadline using the previous so we are not _pushing it_ further
+          pure $ headTxOut & changeHeadOutputDatum (replaceContestationDeadline deadline)
+      , SomeMutation (Just "must not push deadline") PushDeadlineAlthoughItShouldNot <$> do
+          alreadyContested <- vectorOf (length healthyParties - 1) $ Plutus.PubKeyHash . toBuiltin <$> genHash
+          let contester = toPlutusKeyHash $ verificationKeyHash healthyContesterVerificationKey
+          pure $
+            Changes
+              [ ChangeOutput 0 (headTxOut & changeHeadOutputDatum (replaceContesters (contester : alreadyContested)))
+              , ChangeInputHeadDatum (healthyClosedState & replaceContesters alreadyContested)
+              ]
+      , SomeMutation (Just "changed parameters") MutateOutputContestationPeriod <$> do
+          randomCP <- arbitrary
```

Comment:
Could: exclude the `healthyContestationPeriod` to not have a flaky conflict here